dashboards/google-compute-engine/metadata.yaml

```diff
@@ -23,3 +23,11 @@ sample_dashboards:
     related_integrations:
       - id: gce
         platform: GCP
+  -
+    category: Google Compute Engine
+    id: reservations-monitoring
+    display_name: GCE Reservations Monitoring
+    description: "This dashboard enables you to visualize and monitor [local and shared reservation usage](https://cloud.google.com/compute/docs/instances/reservations-monitor) at the owner project level."
+    related_integrations:
```

Comment:
Please remove this section, this is for Google-supported first party dashboards.